.buildkite/nightly.steps.yaml

```
@@ -1,3 +1,15 @@
+script_runner: &script_runner
+  agents:
+  - "agent_count=8"
+  - "capable_of_building=platform"
+  - "environment=production"
+  - "machine_type=quarter"
+  - "permission_set=builder"
+  - "platform=linux"
+  - "queue=${CI_LINUX_BUILDER_QUEUE:-v3-1572524284-e64831bf1e88b227-------z}"
```

```suggestion
  - "queue=${CI_LINUX_BUILDER_QUEUE:-v4-2019-12-12-bk5225-daecba805768d787}"
```
Probably needed, see: https://github.com/spatialos/UnrealGDK/commit/a715e7f9c8cb6c0c10f807632d4b0c5f9287f27d
Also, I just ran an example project build and it's stuck waiting for an agent: https://buildkite.com/improbable/unrealgdkexampleproject-nightly/builds/335